lib/src/nodes/config_node_simple_value.cc

```diff
@@ -0,0 +1,41 @@
+#include <internal/nodes/config_node_simple_value.hpp>
+#include <internal/config_exception.hpp>
+#include <internal/tokens.hpp>
+#include <internal/values/config_string.hpp>
+
+using namespace std;
+
+namespace hocon {
+
+    config_node_simple_value::config_node_simple_value(shared_token value) : _value(move(value)) { }
+
+    shared_token config_node_simple_value::get_token() const {
+        return _value;
```

Comment:
It might be clearer to call it `_token` instead of `_value`. That naming is throwing me off, and doesn't seem to match the java source.